risc0/zkvm/Cargo.toml

```diff
@@ -95,6 +95,7 @@ serial_test = "3.0"
 tar = "0.4"
 tempfile = "3"
 test-log = { version = "0.2", default-features = false, features = ["trace"] }
+risc0-groth16 = { workspace = true, features = ["docker"] }
```

Comment:
We should not se the `features = ["docker"]` here, instead we need to forward the `docker` flag from `risc0-zkvm` to `risc0-groth16.`